mcs/build/profiles/testing_aot_full.make

```diff
@@ -5,6 +5,8 @@ BOOTSTRAP_PROFILE = build
 BOOTSTRAP_MCS = MONO_PATH="$(topdir)/class/lib/$(BOOTSTRAP_PROFILE)$(PLATFORM_PATH_SEPARATOR)$$MONO_PATH" $(INTERNAL_CSC)
 MCS = $(BOOTSTRAP_MCS)
 
+PLATFORMS = darwin linux win32
```

Comment:
For that, you also want to use [this](https://github.com/mono/mono/compare/master...luhenry:corefx-System.Net.NetworkInformation#diff-8a17f52e0062026d1a9555a3bae175e8R298) since `PLATFORMS` is useful when you want to cross-compile to another platform, and there is no point of cross-compiling `testing_aot_full` since it's only for testing.